templates/support.html

```diff
@@ -0,0 +1,160 @@
+{% extends "base_index.html" %}
+
+{% block page_class %}Bare metal support{% endblock %}
+{% block page_title %}Get help for all your bare metal needs with Canonical, from enterprise bare metal support to training and professional services.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1oEF37NP9bDuKQpKBQjUfWleN3yQ3QRUp9d2pLiKryCg/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-takeover--dark">
+  <div class="row">
+    <div class="col-6 u-vertically-center">
+      <h1>Get bare metal-support</h1>
+      <p>Need help? Canonical offers several options for enterprise support, as well as training and professional services.</p>
+      <p>
+        <a class="p-button--positive" href="/contact-us">Contact us</a>
```

Comment:
```suggestion
        <a class="p-button--positive" href="/contact-us">Get in touch</a>
```
Is this in the copydoc